Automattic/jetpack
# Script for preparing release branch

#### Changes proposed in this Pull Request:

Adds a new bash script that will purge dev files, leaving only the ones that are used in production. 

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* New script at least 

#### Testing instructions:






- Run `bin/purge-dev-files.sh` at the root of the Jetpack directory. 
- You should see all the files deleted that are listed in .svnignore

#### Proposed changelog entry for your changes:

* N/A
